src/UnitTest/test_quote_prov.cpp

```diff
@@ -439,6 +613,25 @@ static void GetRootCACrlTest()
     ASSERT_TRUE(TEST_SUCCESS);
 }
 
+static void GetRootCACrlICXTest()
+{
+    boolean TEST_SUCCESS = false;
+
+    char* root_ca_crl = nullptr;
+    uint16_t root_ca_crl_size;
+    quote3_error_t result =
+        sgx_ql_get_root_ca_crl(&root_ca_crl, &root_ca_crl_size);
```

Comment:
`sgx_ql_get_root_ca_crl` build the endpoint URI from constant `ROOT_CA_CRL`. For ICX testing here, do we want to test SBX endpoint to make sure it also works?